bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/FormContainerImpl.java

```diff
@@ -205,4 +208,22 @@ public int getColumnCount() {
     public AllowedComponentsExporter getExportedAllowedComponents() {
         return null;
     }
+
+    @JsonIgnore
+    public Map<String, String> visit(Function<FormComponent, Map<String, String>> callBack) throws Exception {
+        return traverseChild(this, callBack);
+    }
+
+    private Map<String, String> traverseChild(Container container, Function<FormComponent, Map<String, String>> callBack) throws Exception {
+        Map<String, String> result = null;
```

Comment:
result is being set to null every time. this is a restriction for someone who wants to persist some previous field data as well while visiting the form fields.